### DISCUSSION TITLE
v2ex 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#41] [模板讨论]
v2ex 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by 730308009** [comment]
签到报错 又没人知道什么意思的啊...

Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Failed to connect to www.v2ex.com port 443 after 15224 ms: Operation timed out,
Request URL: https://www.v2ex.com/mission/daily

**Reply by wjf0214** [reply]
> 签到报错 又没人知道什么意思的啊...
> 
> Failed at 1/5 request, Fail assert: {"re": "200", "from": "status"} from success_asserts, Response Error : HTTP 599: Failed to connect to [www.v2ex.com](http://www.v2ex.com) port 443 after 15224 ms: Operation timed out, Request URL: [v2ex.com/mission/daily](https://www.v2ex.com/mission/daily)

请求超时。网络问题。v2ex国内网络不能直接访问。

**Comment by miaoxiaozhao** [comment]
用了正确的代理，提示超时。

**Reply by wjf0214** [reply]
应该还是代理问题，我这正常。超时一般就还是网络问题。

**Comment by PrinSoul** [comment]
这个是不是不能用了？

**Reply by wjf0214** [reply]
我这一直可以，没有问题。

**Comment by luyud** [comment]
还能用吗， 换了cookie还是403


```
Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.v2ex.com/mission/daily
```
